### PR TITLE
Add pyrolite

### PR DESCRIPTION
**Pull request by morganjwilliams**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] ~If static libraries are linked in, the license of the static library is packaged.!~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by morganjwilliams** [comment]
Checks are currently failing due to [`pathlib` seeming to be missing](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=310071&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1290) (despite being a standard library package since Py3.4) - I'll have to debug this one, and it might have to wait for the next `pyrolite` PyPI release if the issue is with dependency specification within the package itself.

**Comment by eazzzon** [comment]
this is strange, I will take a look as well

**Comment by morganjwilliams** [comment]
@eazzzon looks like that raises a linting flag and won't solve our issue alone.

I had another look at `pathlib`, and it seems there's a separate `pathlib` (and `pathlib2`) package on PyPI. I'm not well versed in the mechanics of the `conda`  and `pip` dependency resolvers, but perhaps the standard library `pathlib` is being ignored and the standalone package version is expected by `pip check` (which in this case may not actually be installed, depending on `conda`'s dependency resolver and how it handles standard library dependencies...).

**Comment by morganjwilliams** [comment]
Hey @eazzzon - with 0.3.1 released (apologies for the long delay on this one, it's been a busy year!), this looks like it's now good to go. There's a residual issue with using `pip check`, which I think is due to a recent update to how `mpltern` has set up versioning; for now I've left this check out and will look to do some debugging to see if it can be addressed. 

Please post here if you're still happy to be a maintainer for the conda recipe side of things, and we can tick the last box above.

**Comment by eazzzon** [comment]
> Hey @eazzzon - with 0.3.1 released (apologies for the long delay on this one, it's been a busy year!), this looks like it's now good to go. There's a residual issue with using `pip check`, which I think is due to a recent update to how `mpltern` has set up versioning; for now I've left this check out and will look to do some debugging to see if it can be addressed.
> 
> Please post here if you're still happy to be a maintainer for the conda recipe side of things, and we can tick the last box above.

Hi @morganjwilliams, sorry for the late delay, the notification of github is a bit confusing.
Very good news for the recipe, I am very happy to be one of the maintainer.

File: recipes/pyrolite/meta.yaml
Comment (dopplershift) [inline recipes/pyrolite/meta.yaml:48]: Probably better to do:
```suggestion
  license: LicenseRef-CSIRO-Open-Source
```
since there's so much legalese at the bottom.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Minor tweak to the license metadata and this is good to go.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**
